src/LandingPage/LandingPage.css

```diff
@@ -0,0 +1,20 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Poppins:wght@300;400;500;600&display=swap');
```

Comment:
Put it under `index.css` so other components CSS has access to the same font

Reply:
done